gulpfile.js

```diff
@@ -84,46 +84,35 @@ gulp.task('server', () => {
 
 gulp.task('test:app', () =>
   gulp.src(paths.testSpecs + 'app_spec.js', {read: false})
-  .pipe(mocha({reporter: 'spec'}))
+  .pipe(mocha({reporter: 'spec', exit: true}))
   // https://github.com/sindresorhus/gulp-mocha#test-suite-not-exiting
-  .once('error', () => {
-    process.exit(1)
-  })
-  .once('end', () => {
-    process.exit()
-  })
+  .on('error', console.error)
 )
 
-gulp.task('test', cb => {
-  runsequence('build', ['test:app'], cb)
-})
+gulp.task('test', gulp.series(
+  'build',
+  'test:app'
+))
 
 // Watch tasks ----------------------------
 // When a file is changed, re-run the build task.
 // ---------------------------------------
 
-gulp.task('watch:styles', () => {
-  return gulp.watch(paths.assetsScss + '**/*.scss', ['styles'])
-})
-
-gulp.task('watch:scripts', () => {
-  return gulp.watch(paths.assetsJs + '**/*.js', ['scripts'])
-})
-
-gulp.task('watch:images', () => {
-  return gulp.watch(paths.assetsImg + '**/*', ['images'])
-})
-
-gulp.task('watch', ['watch:styles', 'watch:scripts', 'watch:images'])
+gulp.task('watch', () => Promise.all([
+  gulp.watch(paths.assetsScss + '**/*.scss', gulp.task('styles')),
+  gulp.watch(paths.assetsJs + '**/*.js', gulp.task('scripts')),
+  gulp.watch(paths.assetsImg + '**/*', gulp.task('images'))
+]))
 
 // Develop task --------------------------
 // Runs copy-assets task and sets up watches.
 // ---------------------------------------
-gulp.task('develop', cb => {
-  runsequence('build',
-              'watch',
-              'server', cb)
-})
+
+gulp.task('develop', gulp.series(
```

Comment:
Minor, but if I `Ctrl-C` this task I get:

```
[09:30:33] The following tasks did not complete: develop, server
[09:30:33] Did you forget to signal async completion?
```

This doesn't happen on `master`.

Reply:
Thanks, fixed now.